src/main/java/com/redhat/pantheon/model/Product.java

```diff
@@ -0,0 +1,31 @@
+package com.redhat.pantheon.model;
+
+import java.util.Calendar;
+
+import javax.annotation.Nonnull;
+
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.Default;
+
+import com.redhat.pantheon.model.api.SlingResource;
+
+public class Product extends SlingResource {
```

Comment:
Can we have some class javadocs please? :smile:

Reply:
done.